templates/oam/workloads/multiclusterlinkerdaddon.meshery.layer5io.schema.json

```diff
@@ -0,0 +1,7 @@
+{
+    "$id": "http://meshery.layer5.io/definition/Workload/MultiClusterLinkerdAddon",
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "MultiClusterLinkerdAddon",
```

Comment:
The `title` field is supposed to be human readable (friendly), as this is directly displayed in the UI.